File: R/tests/testthat/test-package.R
#!/usr/bin/env Rscript
# Script: test-package.R
# Objective: Verify that the rclt package metadata loads under testthat.
# Author: Yi Yu
# Created: 2026-06-23
# Last updated: 2026-06-23
# Inputs: Installed rclt package metadata.
# Outputs: Testthat assertions.
# Usage: Loaded by testthat::test_local("rclt").
# Dependencies: testthat, rclt

test_that("package metadata is available", {
  expect_equal(as.character(utils::packageVersion("rclt")), "0.1.0")
})

File: python/tests/test_package.py
#!/usr/bin/env python3
"""
Script: test_package.py
Objective: Verify Python CLTF package metadata and public imports.
Author: Yi Yu
Created: 2026-06-24
Last updated: 2026-06-24
Inputs: Installed editable Python package.
Outputs: Pytest assertions.
Usage: python -m pytest python/tests/test_package.py -q
Dependencies: pytest, cltf
"""

import cltf


def test_package_version() -> None:
    assert cltf.__version__ == "0.1.0"
